scripts/globals/mobskills/arcane_stomp.lua

```diff
@@ -20,8 +19,7 @@ function onMobWeaponSkill(target, mob, skill)
 
     local duration = 300;
     local typeEffect = dsp.effect.MAGIC_SHIELD;
-    local msg = MobBuffMove(mob,typeEffect,3,0,duration);
```

Comment:
since you are de-variablizing msg, may as well do the same for duration and effect since they never change either. none of the functionality changes.